tests/test_messages.py

```diff
@@ -185,7 +185,7 @@ def test_validate_payload_with_valid_payload(ocpp_version):
     message = CallResult(
         unique_id="1234",
         action="Heartbeat",
-        payload={'currentTime': datetime.now().isoformat()}
+        payload={'currentTime': datetime.now().isoformat()+'Z'}
```

Comment:
for consistency, I suggest we use the datetime result of this:
`datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S') + "Z"`
which is what we use in ocpp/routing.py

Also, since it is related, maybe you can fix all the fields where we are incorrectly using the datetime and substitute them by the line above